### PR TITLE
🚧Improve release tweet process

### PR DESCRIPTION
**Pull request by CalvinWilkinson**
<h1 style="font-weight:bold" align="center">Preview Feature Pull Request</h1>
<h2 style="font-weight:bold" align="center">📋Review Checklist📋</h2>

- [x] The **_head(source)_** branch for this pull request is a **_preview feature_** branch, with the correct naming syntax, in the following example:
  - A **_head(source)_** branch is the branch being merged into a target branch.
    <details closed><summary>Example</summary>

      ``` xml
      Syntax: preview/feature/<issue-num>-<description>
      Example: preview/feature/123-my-preview-feature
      ```
    </details>

- [x] The **_head(source)_** branch for this pull request is created from a **_preview release_** branch, with the correct naming syntax, in the following example:
  <details closed><summary>Example</summary>

    ``` xml
    Syntax: preview/v<major>.<minor>.<patch>-preview.<prev-num>
    Example: preview/v1.2.3-preview.4
    ```
  </details>

- [x] The **_base(target)_** branch for this pull request is a **_preview release_** branch, with the correct naming syntax, in the following example:
  - A **_base(target)_** branch is the branch that the **_head(source)_** branch is merging into.

    <details closed><summary>Example</summary>

      ``` xml
      Syntax: preview/v<major>.<minor>.<patch>-preview.<prev-num>
      Example: preview/v1.2.3-preview.4
      ```
    </details>

💡For more information on branching, refer to the project's [branching documentation](../../Documentation/Branching.md).

- [x] Associated issue exists and is linked to this pull request.
  - One issue per pull request.

- [x] Pull request title matches the title of the associated issue.

- [x] A `preview` label has been added to the pull request.

- [x] The labels attached to this PR match the labels attached to the associated issue.

- [x] Project assigned matches the project of the associated issue.

- [x] Milestone assigned matches the milestone of the associated issue.

- [x] My code follows the coding style of this project.
  - The style is enforced by the **_.editorconfig_** files in the project and displayed as warnings.  If there is an edge case, reach out, and let's discuss it.

- [x] I have manually tested my code changes to the best of my ability.

- [x] All tests passed locally.
  - This is required because unit tests might pass locally but not in the CI environment during the status check process or vice-versa.
  - Tests might pass on the developer's machine but not necessarily on the code reviewer's machine.
  - Status checks run for this pull request when merging into the **_preview/v\*.\*.\*-preview.\*_** branch.  These status checks run every time a change is pushed to the pull request.  These checks validate version syntax, tagging, builds, unit tests, and more.

### CODE REVIEW

File: CICD/CICD.cs [hunk 1]
```diff
@@ -67,13 +70,27 @@ public partial class CICD : NukeBuild
 
     private Configuration Configuration => GetBuildConfig();
 
-    private IGitRepoService Repo => App.Container.GetInstance<IGitRepoService>();
+    private IGitRepoWrapper Repo => App.Container.GetInstance<IGitRepoWrapper>();
 
     [Parameter("The output directory of where the template workflows should be generated.")]
     private string? WorkflowTemplateOutput { get; set; }
 
     [Parameter("If true, will skip the twitter announcement of a release.")]
```
Comment (kselena) [inline CICD/CICD.cs:78]: Refactor all lowercase twitter to uppercase Twitter.
```suggestion
    [Parameter("If true, will skip the Twitter announcement of a release.")]
```

File: Testing/CICDTests/Services/TweetServiceTests.cs [hunk 2]
```diff
@@ -0,0 +1,357 @@
...
+
+        this.mockFile = new Mock<IFile>();
+        this.mockFile.Setup(m => m.Exists(It.IsAny<string>())).Returns(true);
+
+        this.mockRepoInfoReactable = new Mock<IReactable<(string, string)>>();
+        this.mockSkipTweetReactable = new Mock<IReactable<bool>>();
+    }
+
+    #region Constructor Tests
+
```
Comment (KinsonDigitalAdmin) [inline Testing/CICDTests/Services/TweetServiceTests.cs:55]: Remove empty line after `#region`

**Review by KinsonDigitalAdmin — CHANGES_REQUESTED** [review]

File: CICD/Services/Interfaces/IPullRequestService.cs [hunk 3]
```diff
@@ -2,7 +2,7 @@
 // Copyright (c) KinsonDigital. All rights reserved.
 // </copyright>
 
-namespace CICDSystem.Services;
+namespace CICDSystem.Services.Interfaces;
 
 /// <summary>
 /// Provides information about the current build pull request.
```
Comment (kselena) [inline CICD/Services/Interfaces/IPullRequestService.cs:8]: Grammar
```suggestion
/// Provides information about the current build's pull request.
```

**Review by kselena — CHANGES_REQUESTED** [review]

File: CICD/App.cs [hunk 4]
```diff
@@ -25,7 +26,7 @@ internal static class App
     private static bool isInitialized;
 
     /// <summary>
-    /// Gets the inversion of control container used to get instances of objects.
+    /// Gets the inversion of control container used to produce instances of objects.
```
[the remark this answers is not in the document]
Comment (CalvinWilkinson) [reply CICD/App.cs:29]: This is not meant to be "control the container".  The type of container is an "Inversion Of Control" container.

The word "the" will not work in this case.

The popular acronym for "Inversion Of Control" is **IoC**.  Should I do the example below:

```suggestion
    /// Gets the IoC container used to produce instances of objects.
```

File: CICD/CICD.cs (hunk 1 above)
Comment (CalvinWilkinson) [reply CICD/CICD.cs:78]: Great catch!!

File: CICD/App.cs (hunk 4 above)
[the remark this answers is not in the document]
Comment (kselena) [reply CICD/App.cs:29]: Understood.  I'll resolve.

**Review by kselena — APPROVED** [review]

**Review by KinsonDigitalAdmin — APPROVED** [review]
